pandas/tests/tseries/test_frequencies.py

```diff
@@ -7,6 +7,10 @@
 from pandas import (Index, DatetimeIndex, Timestamp, Series,
                     date_range, period_range)
 
+from pandas._libs.tslibs.frequencies import (_period_code_map, get_rule_month,
```

Comment:
maybe move this to test_libfrequencies as seems to be mostly doing that. (could also do ``pandas/tests/tseries/frequencies/test_libfrequencies.py``)